Source/SvgDocument.cs

```diff
@@ -438,6 +438,16 @@ public void Draw(ISvgRenderer renderer)
             this.Render(renderer);
         }
 
+        /// <summary>
+        /// Renders the <see cref="SvgDocument"/> to the specified <see cref="Graphics"/>.
+        /// </summary>
+        /// <param name="graphics">The <see cref="Graphics"/> to be rendered to.</param>
+        /// <exception cref="ArgumentNullException">The <paramref name="graphics"/> parameter cannot be <c>null</c>.</exception>
+        public void Draw(Graphics graphics)

```

Comment:
Hm, shouldn't you remove the default value from the `size` parameter in the other method in this case?

Reply:
You're right. I should remove the default value, but keep the null conditional operator ("?") operator, which I believe was done to  have "null" as default for this parameter (no conversions in SizeF).
In case someone actually uses the optional parameter with "null", to stay compatible.